src/manipulation.js

```diff
@@ -202,7 +202,10 @@ function domManip( collection, args, callback, ignored ) {
 								jQuery._evalUrl( node.src );
 							}
 						} else {
-							DOMEval( node.textContent.replace( rcleanScript, "" ), doc );
+							if ( node.hasAttribute && node.hasAttribute( "nonce" ) ) {
+								nonce = node.getAttribute( "nonce" );
```

Comment:
As @cnsgithub rightly mentioned, this will no longer work in latest standards. You need to use the property, i.e. `nonce = node.nonce`.